content/07.pathogenesis.md

```diff
@@ -25,4 +25,26 @@ This section would also be great for the introduction of zoonotic diseases which
 
 ### Systems level approaches for understanding SARS-CoV-2 pathogenesis
 
-[Sandipan Ray volunteered to lead this section]
+Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
+These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
+Over the last decade, systems biology approaches are used vastly to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
+Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
+
+The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet. 
+A prominent breakthrough in 2019-nCoV research is accomplished with the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10.1016/S0140-6736(20)30251-8; @doi:10.1038/s41586-020-2012-7]. 
```

Comment:
Two suggestions beyond just some minor flow suggestions:
1. I believe SARS-CoV-2 is the more up-to-date name (per https://www.nature.com/articles/s41564-020-0695-z)
2. These citations are causing a bug, but I think it's just because DOIs that have "(" or ")" sadly cause issues. For future reference, you can convert them to remove these characters at this link: http://shortdoi.org/. I believe it should work with the change below.
```suggestion
A significant breakthrough in SARS-CoV-2 research was achieved through the successful full-length genome sequencing of the pathogen [@doi:10.1038/s41586-020-2008-3; @doi:10/ggjr43; @doi:10.1038/s41586-020-2012-7]. 
```